english/eating-walls/sound-of-birds/hunting/find-shelter.md

```diff
@@ -0,0 +1,11 @@
+Freezing, you wander the wilderness looking for shelter.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.